arch/risc-v/src/common/riscv_cpustart.c

```diff
@@ -41,37 +41,22 @@
 #include "riscv_internal.h"
 #include "chip.h"
 
-#ifdef CONFIG_SMP
-
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 
-#if 0
-#  define DPRINTF(fmt, args...) _err(fmt, ##args)
-#else
-#  define DPRINTF(fmt, args...) do {} while (0)
-#endif
-
-#ifdef CONFIG_DEBUG_FEATURES
-#  define showprogress(c) riscv_lowputc(c)
-#else
-#  define showprogress(c)
-#endif
-
 /****************************************************************************
  * Public Data
  ****************************************************************************/
 
 extern volatile bool g_serial_ok;
```

Comment:
remove g_serial_ok too?

Reply:
Ok, and also removed it from non-smp system.